src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesView/ExternalDataSourcesView.js

```diff
@@ -1,140 +1,222 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
-import { Button, Card, Col, KeyValue, Layout, NoValue, Row } from '@folio/stripes/components';
-import { CustomMetaSection } from '@folio/stripes-erm-components';
+import { Button, Card, Col, Icon, KeyValue, Layout, Modal, ModalFooter, NoValue, Row } from '@folio/stripes/components';
+import { ActionMenu, CustomMetaSection } from '@folio/stripes-erm-components';
 import useDisplayMetaInfo from '../useDisplayMetaInfo';
 
 const ExternalDataSourcesView = ({
   input: { value = {} },
   onDelete,
-  onEdit
+  onEdit,
+  onSave
 }) => {
   const { syncStatus, cursor, lastChecked } = useDisplayMetaInfo(value);
+  const [showConfirmResetSyncStatus, setShowConfirmResetSyncStatus] = useState(false);
 
-  return (
-    <Card
-      data-test-external-data-source-view
-      headerEnd={(
-        <span>
-          {!value.readonly &&
-            <Button
-              buttonStyle="danger"
-              data-test-external-data-source-delete
-              marginBottom0
-              onClick={onDelete}
-            >
-              <FormattedMessage id="stripes-core.button.delete" />
-            </Button>
-          }
+  const difference = new Date().getTime() - new Date(value.lastCheck).getTime();
+  const hours = difference ? Math.floor(difference / 1000 / 60 / 60) : 0;
```

Comment:
Small tweak, could we change this to ```const hours = difference ? Math.floor(difference / (1000 * 60 * 60)) : 0;```
Its slightly clearer (and makes my mathematician heart hurt less :wink: )

Reply:
I think this is a simple enough use case that it's fine for now, but other date libraries can do this kind of difference in a specified unit slightly more cleanly than plain Date(). Don't think it's worth changing for now though

Reply:
> I think this is a simple enough use case that it's fine for now, but other date libraries can do this kind of difference in a specified unit slightly more cleanly than plain Date(). Don't think it's worth changing for now though

can you give an example anyway? I found the example on the web and am interested in better solutions, even if we do not use them here